src/services/utils.ts

```diff
@@ -12,3 +12,23 @@ export function debounce(fn: Function, delay: number) {
 export function stringToNumber(text: string | null) {
   return text?.trim() && !isNaN(Number(text)) ? Number(text) : undefined
 }
+
+export function stringToNumbers(text: string | null, separator: string) {
```

Comment:
Might be helpful to type the return types of these functions, also to facilitate readability (even if the function names are already pretty clear).

Reply:
I find this a bit verbose but why not. Updated the returned types.